Partner Manager 2.16.0 (#200)
mulesoft/docs-partner-manager: modules/ROOT/pages/configure-global-as2-settings.adoc

= Configuring Global AS2 Endpoint Settings

> You can prevent Anypoint Partner Manager from processing requests with expired certificates by following these steps:

Reviewer: ```suggestion
To prevent Anypoint Partner Manager from processing requests with expired certificates:
```

. Access the <host> page.

> . To prevent inbound requests with expired certificates from being processed by Partner Manager:

Reviewer: For this and line 10, suggest making these bullets since they're not sequential (unless they are).
. To prevent:
** Inbound requests...
** Outbound requests...

> . To prevent inbound requests with expired certificates from being processed by Partner Manager:

Reviewer: Can you make these active?
